charters/wot-wg-charter-draft-2019.html

```diff
@@ -178,39 +186,369 @@ <h2>Scope</h2>
+          of privacy.
+        </p>
+
+	<h3>TD Vocabulary</h3>
+	<p>
+          The
+          <a href="http://www.w3.org/TR/wot-thing-description">WoT Thing Description (TD)</a>
+          context includes a core set of vocabulary.
+          This vocabulary can be extended by including additional context files.
+          However, to support new use cases and protocols in a consistent (interoperable) way,
+          especially in cases where JSON-LD processing is not possible,
+          extensions to this core vocabulary may be necessary.
+	</p>
+	<p>
+          Candidates for TD vocabulary extension include 
+          support for Data Schemas consistent with current versions of JSON Schema, 
+          extensions to Data Schemas to support XML and CBOR 
+          (and possibly other structured payload data formats), 
+          support for additional standard metadata such as location or device manufacturer,
+          and better support for default values.
+        </p>
+
+        <h3>Protocol Vocabulary</h3>
+	<p>
+          The
+          <a href="http://www.w3.org/TR/wot-thing-description">WoT Thing Description (TD)</a>
+          specification is augmented by a 
+          <a href="http://www.w3.org/TR/wot-protocol-binding">WoT Protocol Binding</a>
+          document which defines templates for how to use a TD to describe Things
+          using specific concrete protocols.
+          However, individual concrete protocols may require additional vocabulary
+          to define header options and other metadata.
+          These are defined using vocabulary extensions specific to each protocol.
+          Currently only a limited set of protocols are supported in a standard way.  
+          This work item will add support for an additional set of target protocols, 
+          where appropriate in collaboration with the defining standards body.
+        </p>
+        <p>
+          Candidates for extended protocol vocabulary include support 
+          for CoAP(S), CoAP(S)+WS(S), MQTT, and OPC-UA.
+        </p>
+
+        <h3>Smart Home Profile</h3>
+        <p>The
+          <a href="http://www.w3.org/TR/wot-thing-description">WoT Thing Description (TD)</a>
+          provides a general framework for describing the interface to a Thing
+          independent of the network protocol, data encoding, or security mechanism used.  
+          TDs are defined to be extensible since IoT protocols, data encodings, and security mechanisms 
+          are constantly evolving.
+          However, this openness works against interoperability since it is 
+          not clear in advance what technologies must be supported by a given
+          implementation of a TD Consumer, and a consumer can only work with
+          Things supporting the technologies it includes an implementation for.
+        </p>
+        <p>
+          Especially in a consumer context, "plug and play" interoperability is
```

Comment:
I don't think this statement (and profile) should be limited to the consumer context, it is even more important in the industrial context, where typically the number of devices that need to interoperate is much higher.

We should try to involve more members from industrial manufacturers to the WG, to make sure their scenarios and use cases get addressed.

Reply:
One difference between my proposal and yours is that I am thinking of verticals (which have a specific set of protocols), you are thinking along the constrained/unconstrained spectrum.   Both make sense...

Reply:
Anyhow, not saying industrial is not important, it obviously is.   I was thinking of changing the deliverable name to simply "Profiles" and putting multiple vertical profiles into one document (assuming we do have vertical profiles).

Reply:
I reworded this, but it may not be completely in alignment with what you want.  What I wanted to say is that "it just works" is ESPECIALLY relevant in the consumer use case, but is of course highly desirable in the industrial use case also.  To discuss.